### PR TITLE
Setup deployment

### PR DESCRIPTION
**Pull request by williamputraintan**
- Added stack for illumination deployment (Check [infrastructure/readme.md](https://github.com/umccr/illumination-v2/blob/setup-deployment/infrastructure/README.md))
- Added SDK to fetch SecretManager value at ReactApp
- ~~Added stack for proxy-server lambda deployment~~ **(No longer needed  ❌)**

~~Waiting for ICAV2 token secret manager [PR](https://github.com/umccr/infrastructure/pull/206) finalised~~  ✅

Deployed in aws-dev account: https://illumination.dev.umccr.org/ (Some routing bug fix at the following [PR#12](https://github.com/umccr/illumination-v2/pull/12))

### CODE REVIEW

File: infrastructure/README.md [hunk 1]
```diff
@@ -1,14 +1,44 @@
...
+
+Modifying existing resource:
+- Route53 - add A Record to the DNS to access api-gateway
+
+Manual Resource Creation:
+- AWS Certificate Manager - SSL certificate needed for cloudfront in `ap-southeast-2` region
+- CName in Route53 -  To allow the above certificate to be verified via DNS
+- AWS SSM Parameter - Store the above SSL Certificate ARN with parameter name `/illumination/ssl_aps2_cert_arn`  
+
+NOTE: Yes there are 2 SSL certificate created for 2 stacks. Ideally we could create one to be used in both stack. CloudFront need the certificate located in USE1 and ApiGatewayV2 (HttpApi) needed to be located at ASP2.
```
Comment (williamputraintan) [inline infrastructure/README.md]: FYI @reisingerf @victorskl ,

We did had a discussion where we wanted to maintain SSL certificate only in one region which is in USE1 as this the one supported by CloudFront. But ApiGatewayV2 (httpApi), does not have EDGE endpoint feature and need to have the certificate the same region as the gateway. So for that reason we still need 2 region certificate, which explains why the data-portal have the same certificate in both region.

Comment (victorskl) [reply infrastructure/README.md]: Noted, thanks. Guess, this won't be needing anymore; now that v2 endpoint supported CORS.

**Comment by victorskl** [comment]
When you are up for final, pls mark to "Ready for review". Then, will do one more round of eyeballing to PR. Tq.

File: infrastructure/README.md (hunk 1 above)
Comment (williamputraintan) [reply infrastructure/README.md]: Yup!

**Comment by williamputraintan** [comment]
> When you are up for final, pls mark to "Ready for review". Then, will do one more round of eyeballing to PR. Tq.

Yes, waiting for the v2 secret manager to be deployed.

**Pull request merged**